pkg/model/BUILD.bazel

```diff
@@ -59,7 +61,6 @@ go_library(
 go_test(
     name = "go_default_test",
     srcs = ["bootstrapscript_test.go"],
-    data = glob(["tests/**"]),  #keep
```

Comment:
This has to be added back :)

Reply:
But verify-bazel passed?

Reply:
This is manually added and cannot be verified by that, if I recall correctly.